src/test/steps/admin/configuration/socialMediaAuthenStep.ts

```diff
@@ -0,0 +1,51 @@
+import { Given, When, Then } from "@cucumber/cucumber";
+import SocialMediaPage from "../../../pages/admin/configuration/socialMediaAuthenPage";
+import { pageFixture } from "../../../../hooks/pageFixture";
+
+When("User access social media authen page", { timeout: 30000 }, async () => {
```

Comment:
Please set timeout in config for global configuration. https://github.com/cucumber/cucumber-js/blob/main/docs/configuration.md